### DISCUSSION TITLE
halo2.0统计代码在哪里插入

### DISCUSSION THREAD

**Discussion by VapeSeven** [halo-dev/halo#3953] [Q&A]
### 你当前使用的版本

2.5

### 描述一下此特性

halo2.0统计代码在哪里插入

**Comment by ruibaby** [comment]
<img width="827" alt="image" src="https://github.com/halo-dev/halo/assets/21301288/46812f0d-b93e-4d6b-b391-b93169aa2299">

**全局 head 标签** 或者 **页脚** 都可以。

/kind support
